app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListViewModel.kt

```diff
@@ -17,28 +17,33 @@
  */
 package com.infomaniak.mail.ui.main.folder
 
+import android.app.Application
 import android.text.format.DateUtils
+import androidx.lifecycle.AndroidViewModel
 import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
+import androidx.webkit.WebViewCompat
 import com.infomaniak.mail.di.IoDispatcher
 import com.infomaniak.mail.utils.SearchUtils
 import com.infomaniak.mail.utils.coroutineContext
+import com.infomaniak.mail.utils.extensions.appContext
 import dagger.hilt.android.lifecycle.HiltViewModel
 import kotlinx.coroutines.*
 import javax.inject.Inject
 
 @HiltViewModel
 class ThreadListViewModel @Inject constructor(
+    application: Application,
     private val searchUtils: SearchUtils,
     @IoDispatcher private val ioDispatcher: CoroutineDispatcher,
-) : ViewModel() {
+) : AndroidViewModel(application) {
```

Comment:
No need to turn the view model into an AndroidViewModel, you can directly inject the app context with Hilt like in other view models